CMakeLists.txt

```diff
@@ -130,6 +130,11 @@ set(TILEDB_INSTALL_LIBDIR "" CACHE STRING "If non-empty, install TileDB library
 # Superbuild setup
 ############################################################
 
+# capnpproto requires C++14
+if (TILEDB_SERIALIZATION)
+  set(CMAKE_CXX_STANDARD 14)
```

Comment:
If a build configured with TILEDB_SERIALIZATION has to compile with 14, should this just become globally applicable since rest of build will have to function with that level anyway?

Reply:
This applies globally for serialization builds. If we bump unconditionally for all builds, we should go up to C++17 IMO, but probably need to have a separate discussion about that.